cmake/bintools/llvm/target.cmake

```diff
@@ -11,8 +11,15 @@ find_program(CMAKE_AR      llvm-ar      ${find_program_clang_args}   )
 find_program(CMAKE_NM      llvm-nm      ${find_program_clang_args}   )
 find_program(CMAKE_OBJDUMP llvm-objdump ${find_program_clang_args}   )
 find_program(CMAKE_RANLIB  llvm-ranlib  ${find_program_clang_args}   )
-find_program(CMAKE_OBJCOPY objcopy      ${find_program_binutils_args})
-find_program(CMAKE_READELF readelf      ${find_program_binutils_args})
+if ("${ARCH}" STREQUAL "arm")
+  find_program(CMAKE_OBJCOPY ${triple}-objcopy PATH ${ZEPHYR_SDK_INSTALL_DIR}/${triple}/bin NO_DEFAULT_PATH)
+  find_program(CMAKE_READELF ${triple}-readelf PATH ${ZEPHYR_SDK_INSTALL_DIR}/${triple}/bin NO_DEFAULT_PATH)
```

Comment:
Would it be better to use `${CROSS_COMPILE}` as that could also remove the `if ... else` when building for host.
https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/bintools/gnu/target.cmake#L5-L6

or is there some special case in this situation where we are combining LLVM/clang with the GNU bintools ?